test/suite/test_oligarch09.py

```diff
@@ -66,12 +72,13 @@ def conn_extensions(self, extlist):
         DisaggConfigMixin.conn_extensions(self, extlist)
 
     def test_oligarch_read_write(self):
-        uri = "oligarch:test_oligarch09"
         create_session_config = 'key_format=S,value_format=S,block_compressor={}'.format(self.block_compress)
+        if self.uri.startswith("file"):
```

Comment:
Can you refactor this by adding a method that generates the config string for the corresponding URI (passed in as a parameter)?

Reply:
Good call! Thanks, done in 37dcc79.